api/types/constants.go

```diff
@@ -662,6 +662,11 @@ const (
 	// ReqAnnotationNotifyServicesLabel is the request annotation key at which notify services are stored for access plugins.
 	ReqAnnotationNotifyServicesLabel = "/notify-services"
 
+	// ReqAnnotationApprovedCloseCode is the request annotation key at which the servicenow close code for approved access requests is stored, this label is added to roles to determine what close code to use when closing the ServiceNow incidents created for access requests. ServiceNow close codes are set on the requesting role.
+	ReqAnnotationApprovedCloseCode = "/approved-close-code"
+	// ReqAnnotationDeniedCloseCode is the request annotation key at which the servicenow close code for denied access requests is stored, this label is added to roles to determine what close code to use when closing the ServiceNow incidents created for access requests. ServiceNow close codes are set on the requesting role.
+	ReqAnnotationDeniedCloseCode = "/denied-close-code"
```

Comment:
@EdwardDowling Can you clarify what these are for? Why would a user need to set these on the roles? Seems confusing.

Reply:
Changing this so close code will be configured at the plugin level instead of per role